src/store/AppStore.js

```diff
@@ -1,10 +1,12 @@
 import { types } from 'mobx-state-tree'
 import { MLTaskStore } from './MLTaskStore'
+import { MLResultsStore } from './MLResultsStore'
 
 const AppStore = types.model('AppStore', {
   
   user: types.optional(types.string, 'Anonymous'),  
-  mlTask: types.optional(MLTaskStore, {})  // We can use {} to set the initial values of a store
+  mlTask: types.optional(MLTaskStore, {}),  // We can use {} to set the initial values of a store
```

Comment:
I think in more recent versions this is `types.safeReference(MLTaskStore)`.

Reply:
If this instantiating a modal as a child node, then this will work or you can pass a function as here: https://github.com/zooniverse/front-end-monorepo/blob/20106662d553811e0fd63fbe4d0aea29437c4576/packages/lib-classifier/src/store/RootStore.js#L16-L30

The code here will default to empty object, where as the version in the monorepo will default to the store that is given an empty object as a snapshot.

`safeReference` is used if you want this to be a reference to another type, typically an array, map, or another model that can be invalidated: https://github.com/mobxjs/mobx-state-tree#reference-validation-isvalidreference-tryreference-oninvalidated-hook-and-typessafereference 

It'll make this a getter/setter function, not the model directly. I'm not sure which is more appropriate here, but I just wanted to clarify the difference and that `types.optional(Model, {})` still is valid.

Reply:
Thanks Sarah and Jim. 👍 The clarification of ways to instantiate a child node/sub-store has been very useful.

For this PR, I'll keep the `types.optional(Model, {})` implementation (since it chains into the next PR), but I want to revisit the monorepo's `types.optional(Model, () => Model.create({}))` implementation in a future PR. The '.create()' pattern seems more promising for more complex stores.